src/makeRobotAccountant.js

```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return x => {
+    return y => {
```

Comment:
better always wrap your params with brackets `()`
```suggestion
  return (x) => {
    return (y) => {
```